### ISSUE TITLE
Preprocessing Issue

### ISSUE THREAD

**Issue by ting7943** [j-morano/rrwnet#13]
Hello, I would like to ask whether the correct order is to first perform label reconstruction, so that the non-vascular regions in the mask are excluded from training, and then apply image preprocessing. I noticed that in the training set, the _data/enhanced_masks actually uses the original masks, rather than masks where the non-vascular regions have been explicitly excluded.
Thanks!

**Comment by j-morano** [comment]
Hello. Thanks for your question. Not sure if I understand the issue. The only preprocessing needed for training are the normalization of the images and obtaining the masks of the region of interest (ROI). In the `enhanced_masks`, only the ROI is set to one, so the other parts are not used for computing the loss.

**Comment by ting7943** [comment]
Hello.I processed the dataset using data_processing/groundtruth.py to reconstruct the labels and separate the non-vascular regions, since I saw in your previous paper that the non-vascular regions do not participate in training.After that, I used the generated masks together with the original images and applied the preprocessing in preprocessing.py. The resulting images look like the example shown in the figure.

May I ask if I did something incorrectly? Because the results look a bit different from the images used in your training.
Thanks！

<img width="1410" height="680" alt="Image" src="https://github.com/user-attachments/assets/37768fc6-0f45-4524-966d-78f1dbf6b2dc" />

**Comment by j-morano** [comment]
While that would probably work, in the current version of the code, that is not needed anymore, as the exclusion of certain areas is done dynamically in the loss function using the ground truth. Thus, the only preprocessing needed is for the image itself and for the ROI mask (with no vessel exclusion).

**Comment by ting7943** [comment]
Thank you very much for your reply! It has cleared up my confusion.
Wish you a pleasant day!

**Comment by j-morano** [comment]
No problem! Thank you for reaching out.
